app/browser/select-dataset-step.tsx

```diff
@@ -143,8 +143,15 @@ const prepareSearchQueryFilters = (filters: BrowseFilter[]) => {
           d.__typename !== SearchCubeFilterType.DataCubeAbout
       )
       .map((d) => {
-        const type = SearchCubeFilterType[d.__typename];
-        return { type, label: d.label, value: d.iri };
+        const type =
+          d.__typename === "Termset"
+            ? SearchCubeFilterType.Termset
+            : SearchCubeFilterType[d.__typename];
```

Comment:
Wouldn't `const type = SearchCubeFilterType[d.__typename]` work here?

Reply:
It does now, thanks I missed it.